config/resources/publicatie-domain.json

```diff
@@ -96,10 +96,17 @@
           "target": "identification",
           "cardinality": "many"
         },
-        "subcases": {
-          "predicate": "ext:doorloopt",
-          "target": "subcase",
-          "cardinality": "many"
+        "translation-subcase": {
+          "predicate": "pub:doorlooptVertaling",
+          "comment": "FIXME: predicate should be dossier:doorloopt",
+          "target": "translation-subcase",
+          "cardinality": "one"
+        },
+        "publication-subcase": {
+          "predicate": "pub:doorlooptPublicatie",
+          "comment": "FIXME: predicate should be dossier:doorloopt",
```

Comment:
Ik denk dat Sven voor de key `_comment` gegaan is

Reply:
Okido. `_comment` it is then :+1: